crates/re_types_builder/src/lib.rs

```diff
@@ -329,6 +332,42 @@ pub fn generate_python_code(
     objects: &Objects,
     arrow_registry: &ArrowRegistry,
 ) {
+    re_tracing::profile_function!();
     let mut gen = PythonCodeGenerator::new(output_pkg_path.as_ref());
     let _filepaths = gen.generate(objects, arrow_registry);
 }
+
+pub(crate) fn rerun_workspace_path() -> camino::Utf8PathBuf {
+    let workspace_root = if let Ok(manifest_dir) = std::env::var("CARGO_MANIFEST_DIR") {
+        let manifest_dir = camino::Utf8PathBuf::from(manifest_dir);
+        manifest_dir
+            .parent()
+            .unwrap()
+            .parent()
+            .unwrap()
+            .to_path_buf()
+    } else {
+        let file_path = camino::Utf8PathBuf::from(file!());
+        file_path
+            .parent()
+            .unwrap()
+            .parent()
+            .unwrap()
+            .parent()
+            .unwrap()
```

Comment:
Is there really no better way to do this? Maybe something like: `file_path.join("../../..")`?